packages/wrangler/src/pages/dev.ts

```diff
@@ -251,12 +251,7 @@ export const Handler = async (args: PagesDevArguments) => {
 		logger.loggerLevel = args.logLevel;
 	}
 
-	/*
-	 * `unstable_dev` already does this for us, but ideally we'd want to print
-	 * the banner right at the top, before any other logs. So let's turn that
-	 * off, and print here instead.
-	 */
-	await printWranglerBanner(true);
```

Comment:
`pages dev` is the only place where we call `printWranglerBanner(true);`. In the spirit of consistency, changing this to `printWranglerBanner();`, which basically mean, we won't check whether a new wrangler version is available, and will just display the currently installed version